Modules/core/Controller/CoresecureController.php

```diff
@@ -79,7 +79,7 @@ protected function checkRememberMeCookie() {
      */
     public function runAction($module, $action, $args = array()) {
         $modelConfig = new CoreConfig();
-        if ($modelConfig->getParam("is_maintenance") && ($this->request->getSession()->getAttribut("user_status") < 4)) {
+        if ($modelConfig->getParam("is_maintenance") && ($this->request->getSession()->getAttribut("user_status") < 2)) {
```

Comment:
rather than 2, 4, should use existing variables (easier to read understand) such as CoreStatus::$ADMIN

Reply:
but there are different values among CoreSpace, CoreStatus and CoreUser.
CoreSpace is for user roles in space, but CoreStatus:$ADMIN vs CoreUser:$ADMIN ???